### PR TITLE
add `--print` mode

### PR DESCRIPTION
**Pull request by juliangruber**
Deyoloify the project _just a bit_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,14 @@ yolox "extract audio from maths.mp4 and save it as maths.m4a" --model=llama3
 # ffmpeg -i maths.mp4 -vn -acodec copy maths.m4a
 ```
 
+### Print mode
+
+Print the command but don't execute it:
+
+```console
+yolox --print "extract audio from maths.mp4 and save it as maths.m4a"
+# ffmpeg -i maths.mp4 -vn -acodec copy maths.m4a
```
Comment (zeke) [inline README.md:90]: I'm into this idea!

But lets change the output to not have a `#` comment at the start. That way you can copy the full output, or pipe it into a file, or pipe it into bash, without having the tweak it.

Comment (juliangruber) [reply README.md]: It already doesn't include `#`, I just thought to make the documentation consistent with other examples. But you're right that here it's misleading 👍

**Review by zeke — APPROVED** [review]
Looks good to me but there are some conflicts. Can you rebase?

**Comment by juliangruber** [comment]
done :)

**Comment by zeke** [comment]
```
$ npx np minor

  ✔ Prerequisite check
  ✔ Git
  ✔ Installing dependencies using npm
  ✔ Running tests
  ✔ Bumping version
  ✔ Publishing package
  ✔ Pushing tags
  ✔ Creating release draft on GitHub

 yolox 3.1.0 published 🎉
```

**Comment by zeke** [comment]
Thanks @juliangruber 🙌🏼

**Pull request merged**